helm/botkube/values.yaml

```diff
@@ -202,7 +202,19 @@ config:
     enabled: false                               # Set to true and specify cert path in the next line after uncommenting
     #cert:                                       # SSL Certificate file e.g certs/my-cert.crt
 
-  # Channels configuration
+  # Setting to support multiple clusters
+  settings:
+    # Cluster name to differentiate incoming messages
+    clustername: not-configured
+    # Set true to enable kubectl commands execution
+    allowkubectl: false
+    # Set true to enable config watcher
+    configwatcher: true
+    # Set false to disable upgrade notification
+    upgradeNotifier: true
+
+communicationConfig:
+  # Communication settings
   communications:
```

Comment:
`communicationConfig.communications` seems to be redundant. We can remove `communicationConfig` and move `communications` at root level

Reply:
@PrasadG193 I have updated the PR. PTAL, I am not sure if this is the correct approach for this.